version/TAG_VERSION

```diff
@@ -1 +1 @@
-0.15.0-beta.2
\ No newline at end of file
+v0.20.0-beta.1
```

Comment:
Just confirmed that this cannot start with `v` (but the tag will get a `v` prefix).  So there's room for improvement in https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/blob/master/dev/bots/projectbots/tag-release#L26 :-)  Probably it's easiest to just add a precommit test that verifies the format though.

Are you OK to remove the `v`?  Then this LGTM...

Reply:
Yes, let's!
```suggestion
0.20.0-beta.1
```